docs/public/specifications/tests/Address-TP/address02.md

```diff
@@ -1,46 +1,97 @@
 ## ADDRESS02: Reverse DNS entry exists for name server IP address
```

Comment:
Change to
```
# ADDRESS02: Reverse DNS entry exists for name server IP address
```

See https://github.com/zonemaster/zonemaster/blob/master/docs/internal/templates/specifications/tests/Template01.md